web/src/reload.js

```diff
@@ -14,11 +14,13 @@ import * as message_lists from "./message_lists";
 import * as narrow_state from "./narrow_state";
 import {page_params} from "./page_params";
 import * as reload_state from "./reload_state";
-import * as server_events from "./server_events";
 import * as ui_report from "./ui_report";
 import * as util from "./util";
 
 // Read https://zulip.readthedocs.io/en/latest/subsystems/hashchange-system.html
+
+let server_cleanup_event_queue;
```

Comment:
Let's make this a `reload_hooks` structure with an `add_reload_hook` method; I don't think we often want to write a function substitution variable like this.